### PR TITLE
Target offers based on profile data

### PR DESCRIPTION
**Pull request by jfkhoury**
## Description

Added a sandbox page that implements sending Target Profile attribute via the `data` field in `sendEvent`, and qualify for an activity based on the visitor profile.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

**Review by ninaceban — APPROVED** [review]
looks great!
I was thinking that maybe we just add button/buttons on the `/personalization` page that would trigger `sendEvent` with profile data, one button would get and render offers for shirt and another for shoes.

File: sandbox/src/useSendPageViewEvent.js [hunk 1]
```diff
@@ -12,11 +12,17 @@ governing permissions and limitations under the License.
...
+  instanceName = "alloy",
+  viewName,
+  data = {},
+  xdm = {}
+} = {}) => {
   useEffect(() => {
-    const xdm = {};
     xdm.eventType = "page-view";
 
+    xdm.identityMap = {};
```
Comment (Aaronius) [inline sandbox/src/useSendPageViewEvent.js:24]: Is this necessary?

**Review by Aaronius — APPROVED** [review]
🏅

**Comment by jfkhoury** [comment]
> looks great!
> I was thinking that maybe we just add button/buttons on the `/personalization` page that would trigger `sendEvent` with profile data, one button would get and render offers for shirt and another for shoes.

I like this idea, I will either make a separate PR or Shamiul can do it in his Sandbox re-write.

File: sandbox/src/useSendPageViewEvent.js (hunk 1 above)
Comment (jfkhoury) [reply sandbox/src/useSendPageViewEvent.js]: Removed ;)

**Pull request merged**